vowpalwabbit/v_array.h

```diff
@@ -352,136 +323,8 @@ struct v_array<T, typename std::enable_if<std::is_trivially_copyable<T>::value>:
     if (_end == _end_array) reserve_nocheck(2 * capacity() + 3);
     new (_end++) T(std::forward<Args>(args)...);
   }
-
-  VW_DEPRECATED("Use std::lower_bound instead. This will be removed in VW 9.0.")
```

Comment:
didn't we just deprecate this stuff a few weeks ago?

Reply:
Yes, prior to the 8.11 release. That is the last release this function will be in.